dockerfiles/ubuntu/Dockerfile

```diff
@@ -0,0 +1,195 @@
+FROM ubuntu:bionic AS resource
+
+
+RUN apt-get update && apt-get install -y \
+  curl \
+  git \
+  gnupg \
+  gzip \
+  jq \
+  openssl \
+  libssl-dev \
+  make \
+  g++ \
+  openssh-client \
+  libstdc++6 \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
+RUN apt-get install -y git-lfs
+
+WORKDIR /root
+RUN git clone https://github.com/proxytunnel/proxytunnel.git; \
+    cd proxytunnel; \
+    make; \
```

Comment:
A little optimization (assuming most of the times the build will be built in a machine that has multiple cores available), as their build makes use of a list of dependencies https://github.com/proxytunnel/proxytunnel/blob/f7831a292f7182dc20e5dbd11a90c18474b6077b/Makefile#L82-L84, we can specify a concurrency factor to `make` to speed the build up with - `make -j4` would do it